src/FSharpPlus/Lens.fs

```diff
@@ -112,6 +112,39 @@ module Lens =
     /// Lens for the fifth element of a tuple
     let inline _5 f t = Map.InvokeOnInstance (fun x -> mapItem5 (fun _ -> x) t) (f (item5 t))
 
+    [<RequireQualifiedAccess>]
+    module List=
+        let removeAt i lst =
+            if List.length lst > i then
+                lst.[0..i-1] @ lst.[i+1..]
+            else lst
+
+        let setAt i x lst =
+            if List.length lst > i then
+                lst.[0..i-1] @ x::lst.[i+1..]
+            else lst
```

Comment:
These 2 functions should go into the List.fs file in Extensions

Reply:
Moved and annotated